GeekDatabase/Course_Enrollment.sql

```diff
@@ -1,14 +1,10 @@
 ﻿CREATE TABLE [dbo].[Course_Enrollment] (
-    [Course_Enrollment_Id]  INT NOT NULL IDENTITY,
     [Profile_Id]             INT NOT NULL,
     [Course_Id]              INT NOT NULL,
-    [Is_Enrollment_Approved] BIT NOT NULL,
```

Comment:
Hát ezzel nem jelentkezést tudnál letiltani, hanem egy konkrét jelentkezést disapprovolni. Szerintem nincs rá nagy szükség, megbeszéléses alapon lesz hogy ki jön mire, nem azon múlik hogy itt be tudja jelölni vagy nem. Törölni meg majd fogunk tudni ha véletlenül jeletkezne vki